geomesa-arrow/geomesa-arrow-gt/src/main/scala/org/locationtech/geomesa/arrow/io/DeltaWriter.scala

```diff
@@ -289,16 +269,38 @@ object DeltaWriter extends StrictLogging {
           val name = fromVector.getField.getName
           val toVector = result.underlying.getChild(name)
           val transfer: (Int, Int, java.util.Map[Integer, Integer]) => Unit =
```

Comment:
Just to make sure I'm understanding....

Dictionaries are paired up IntVectors.  So List[String] turns into a Dictionary + a ListVector[Int].  Further (assertion) since this is about transfers between dictionary-encoded columns, the only cases to worry with are IntVectors and List[Int]Vectors.

Is that right?

Reply:
correct!